### PR TITLE
[SBOM] Enable SBOM in default configuration

### PR DESCRIPTION
**Pull request by derekwbrown**
Enables the SBOM configuration in the configuration template.

### Motivation

Provide configuration documentation for new feature.

### Describe how to test/QA your changes

Install a fresh version of the agent (not an upgrade).
Check the default configuration, that the `datadog.yaml` includes the `SBOM` section, and the other configs (`system-probe`, `security-agent`, etc.) do NOT include the SBOM section

### CODE REVIEW

File: pkg/config/config_template.yaml [hunk 1]
```diff
@@ -1708,6 +1710,28 @@ api_key:
...
+  ## @param enabled - boolean - optional - default: false
+  ## set to true to enable Cloud Security Management Vulnerability Management
+  # enabled: false
+
+  ## uncomment the sections below to enable where the vulnerability scanning is done
+
+  ## @param host
+  ## enable the host-level check
+  # host:
+  #  enabled: false
```
Comment (lebauce) [inline pkg/config/config_template.yaml:1727]: ## @param enabled - boolean - optional - default: false
## set to true to enable Infrastructure Vulnerabiltilies

Comment (shanelhuang) [reply pkg/config/config_template.yaml:1727]: thanks @lebauce added in your suggestions

**Review by cswatt — APPROVED** [review]

**Review by paulcacheux — APPROVED** [review]
LGTM, I think it would be great to have @lebauce 's approval as well

**Review by lebauce — APPROVED** [review]

**Review by jeremy-hanna — APPROVED** [review]
👍

**Pull request merged**